src/components/molecules/AdminManagementForm.tsx

```diff
@@ -0,0 +1,188 @@
+  createStyles({
+    root: {
+      // styles can be added here
+    },
+  }),
+)(TableRow);
+const AdminManagementForm: React.FC<IProps> = ({ saveEditModeHelper, isShowing, onClose, loading }) => {
+  const store: RootStore = useStore();
+  const [editObject, setEditObject] = React.useState<IeditObjList>({});
+
+  const { t } = useTranslation();
+  const classes = useStyles();
+  const EditMode: any = (props: any) => {
+    return (
+      <ButtonGroup>
+        <Button variant="contained" onClick={() => saveEditMode(props.itemData)}>
+          {t('usersManagement.save')}
+        </Button>
+        <Button variant="contained" onClick={() => cancelEditMode(props.itemData)}>
+          {t('usersManagement.cancel')}
+        </Button>
+      </ButtonGroup>
+    );
+  };
+  const ViewMode = (props: any) => {
+    return (
+      <Button onClick={() => changeEditObject(props.itemData)} variant="contained">
+        {t('usersManagement.edit')}
+      </Button>
+    );
+  };
+  const OrganizationEditList: any = ({ itemOrg, itemEmail }: any) => {
+    const orgName = editObject[itemEmail]['organizationValue'] ? editObject[itemEmail]['organizationValue'] : itemOrg;
+    const [currentOrganization, setcurrentOrganization] = React.useState(orgName);
+    const handleChange = (e: any) => {
+      setcurrentOrganization(e.target.value);
+      const newObj = changeCurrentSelectedRoleHelper({ ...editObject }, itemEmail, e.target.value);
+      setEditObject(newObj);
+    };
+    return (
+      <Box>
+        <FormControl fullWidth>
+          <InputLabel id="demo-simple-select-label">{t('userDetailsForm.Organization')}</InputLabel>
+          <Select
+            labelId="demo-simple-select-label"
+            name={itemEmail}
+            value={currentOrganization}
+            label="Organization"
+            onChange={handleChange}
+          >
+            {store.organizationsList &&
+              store.organizationsList.map((org: any) => {
+                return (
+                  <MenuItem key={org} value={org}>
+                    {org}
+                  </MenuItem>
+                );
+              })}
+          </Select>
+        </FormControl>
+      </Box>
+    );
+  };
+  const cancelEditMode = (el: any) => {
+    const newObj = cancelEditModeHelper(el.email, { ...editObject });
+    setEditObject(newObj);
+  };
+  const saveEditMode = (element: any) => {
+    const newObj = saveEditModeHelper(element.email, element.org, { ...editObject });
+    setEditObject(newObj);
+  };
+  const changeEditObject = (element: any) => {
+    const newObj = changeEditObjectHelper(element, { ...editObject });
+    setEditObject(newObj);
+  };
+  return (
+    <DialogWithHeader
+      fullWidth
+      maxWidth={'sm'}
+      isShowing={isShowing}
+      title={t('usersManagement.title')}
+      onClose={() => {
+        onClose();
+      }}
+    >
+      <TableContainer style={{ maxHeight: 250, paddingBottom: 30 }}>
```

Comment:
try to avoid inline styling (`style attribute`), can you put an extra class  on `makeStyles`?